### PR TITLE
feat: review expands and data usage at actitivy list

### PR DESCRIPTION
**Pull request by syoungfntech**
https://tipit.avaza.com/project/view/360253#!tab=task-pane&task=3641047

### CODE REVIEW

File: src/components/tables/room-occupancy-table/OccupancyTable.js [hunk 1]
```diff
@@ -4,10 +4,10 @@ import OccupancyTableHeading from "./OccupancyTableHeading";
 import OccupancyTableCell from "./OccupancyTableCell";
 import OccupancyTableRow from "./OccupancyTableRow";
 import OccupancyActionsTableCell from "./OccupancyActionsTableCell";
+import { TWO } from "../../../utils/constants";
```
Comment (smarcet) [inline src/components/tables/room-occupancy-table/OccupancyTable.js:7]: why this file is changed ?
please roll it back if doe not belongs to the fix

File: src/reducers/events/room-occupancy-reducer.js [hunk 2]
```diff
@@ -101,7 +101,9 @@ const roomOccupancyReducer = (state = DEFAULT_STATE, action) => {
         payload.response.hasOwnProperty("data") &&
         payload.response.data.length === 1
       ) {
-        payloadEvent = payload.response.data[0];
+        const { response: data } = payload;
```
Comment (smarcet) [inline src/reducers/events/room-occupancy-reducer.js:104]: please roll this back if not needed
add more lines of codes

Comment (syoungfntech) [reply src/reducers/events/room-occupancy-reducer.js:104]: This was an eslint requirement, it wants destructuring

File: src/components/tables/room-occupancy-table/OccupancyTable.js (hunk 1 above)
Comment (syoungfntech) [reply src/components/tables/room-occupancy-table/OccupancyTable.js:7]: This was an eslint requirement

**Review by smarcet — CHANGES_REQUESTED** [review]
@syoungfntech please review comments

File: src/actions/event-actions.js [hunk 3]
```diff
@@ -547,11 +547,11 @@ export const getEvents =
 
     const params = {
       expand:
-        "speakers,type,created_by,track,sponsors,selection_plan,location,tags,media_uploads,media_uploads.media_upload_type,actions,actions.type",
+        "speakers,type,created_by,track,sponsors,selection_plan,location,tags,media_uploads,media_uploads.media_upload_type,actions",
       relations:
```
Comment (smarcet) [inline src/actions/event-actions.js:551]: review following relations
  "feedback": [],
            "current_attendance": [],
            "slides": [],
"public_comments": [],
"links": [],
"videos": [],
"media_uploads": [],
"extra_questions": [],
"actions": [],
i dont think they are needed at presentation level

Comment (smarcet) [reply src/actions/event-actions.js:551]: also review sponsors:[] relations
its retrieving a lot of unneeded data ( you probably just need id and name )
 "sponsors": [
                {
                    "id": 5,
                    "name": "Meta",
                    "sponsorships": [
                        1311,
                        1307,
                        1312,
                        1310,
                        1313,
                        1956,
                        1775,
                        2118,
                        1864,
                        1735,
                        1901,
                        1736,
                        2085,
                        1865,
                        2096,
                        1772,
                        1782,
                        1954,
                        1871,
                        1785,
                        1873,
                        1847,
                        1791,
                        1870,
                        1866,
                        1835,
                        1908,
                        1839,
                        1921,
                        1860,
                        1923,
                        1796,
                        1913,
                        1914,
                        1789,
                        1964,
                        1779,
                        1786,
                        1992,
                        1841,
                        1665,
                        1721,
                        1880,
                        2093,
                        1829,
                        1776,
                        1741,
                        1931,
                        1962,
                        3208,
                        3207,
                        3093,
                        3066,
                        3560,
                        4332,
                        4456,
                        4307,
                        5029,
                        6025,
                        6054
                    ],
                    "project_sponsorships": []
                }
            ],

**Review by smarcet — CHANGES_REQUESTED** [review]
@syoungfntech please review comments

**Comment by smarcet** [comment]
closing this one in favor of https://github.com/fntechgit/summit-admin/pull/493

**Pull request closed without merging**